dist/index.html

```diff
@@ -13,12 +13,12 @@
 <body>
```

Comment:
- [ ] OPTIONAL
Your application is not able to run using the command `npm start`. Kindly add the necessary _scripts options_ to correct this issue.

Error generated on the command `npm start`:
![image](https://user-images.githubusercontent.com/12745474/134495319-564c8756-b90f-4f9e-aa0a-791158179222.png)

Reply:
The reason your script isn't working is that I don't have npm start in my package.json. You should have checked so you would know to run `npm run dev` or `npm run build.`